build/build-extensions.ps1

```diff
@@ -52,7 +53,7 @@ function BuildRuntime([string] $targetRid, [bool] $isSelfContained) {
       $suffixCmd = "/p:VersionSuffix=$suffix"
     }
 
-    $cmd = "publish", ".\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj", "-r", "$targetRid", "--self-contained", "$isSelfContained", "/p:PublishReadyToRun=true", "/p:PublishReadyToRunEmitSymbols=true", "-o", "$publishTarget", "-v", "m", "/p:BuildNumber=$buildNumber", "/p:IsPackable=false", "/p:CommitHash=$commitHash", "-c", "Release", $suffixCmd
+    $cmd = "publish", "..\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj", "-r", "$targetRid", "--self-contained", "$isSelfContained", "/p:PublishReadyToRun=true", "/p:PublishReadyToRunEmitSymbols=true", "-o", "$publishTarget", "-v", "m", "/p:BuildNumber=$buildNumber", "/p:IsPackable=false", "/p:CommitHash=$commitHash", "-c", "Release", $suffixCmd
```

Comment:
For this change of location for the publish command, we just need to make sure the script continues to work as expected both local and in Azure DevOps.

Reply:
@madelinegordon -- where are you running the command from? Do you `cd` down into the `build` folder first? I think our CI runs it from the root of the repo which could explain the difference.

Either way, I agree that we should make this work no matter what. Can we get the path of the script, then join it with this path to get the absolute path? I see we have `$rootDir` already in this script.

Reply:
@brettsam Yes, I ran the script from the build folder.